src/bin/fish.rs

```diff
@@ -366,6 +513,21 @@
             'f' => opts.features = w.woptarg.unwrap().to_owned(),
             'h' => opts.batch_cmds.push("__fish_print_help fish".into()),
             'i' => opts.is_interactive_session = true,
+            'I' => {
+                let noconfirm = match w.woptarg {
+                    None => false,
+                    Some(n) if n == L!("noconfirm") => true,
+                    _ => {
+                        FLOGF!(
+                            error,
+                            "Unknown argument to --install: '%ls'",
+                            w.woptarg.unwrap()
+                        );
```

Comment:
I believe we should avoid `FLOG!` flog for user errors that can realistically happen (for consistency and also translations).
So I'd probably use `eprintf!` or `eprintln!` here

Reply:
I disagree - FLOG! is how we print errors all over the place.

See e.g. all the history errors, which can and do happen.

The only reason I'm using eprintln et al at all here is because they're used before initialization is done.